usagov/usagov-2021
# USAGOV-1726: USAGOV-1726 - Exclude taxonomy and åes/taxonomy director…

Exclude taxonomy and /es/taxonomy directory from static site


## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-

## Description
Updated `settings.php` to prevent static-site directories like `html/es/taxonomy` and `html/taxonomy` from being created.

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions
Run Tome and then confirm there are no `html/es/taxonomy` nor `html/taxonomy` directories.
Before this code-change, I was already not seeing a `html/taxonomy` directory, but was able to reproduce the issue with the `html/es/taxonomy` appearing. And this code-change fixes that.

### Change Requirements



### Validation Steps

- Test instruction 1
- Test instruction 2
- Test instruction 3

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
